posts/2020-12-31-Rust-1.49.0.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title: "Announcing Rust 1.40.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.49.0. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.49.0 is as easy as:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.49.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1490-2020-12-31
+
+## What's in 1.49.0 stable
+
+For this release, we have some new targets and an improvement to the test
+framework. See the [detailed release notes][notes] to learn about other
+changes not covered by this post.
+
+### aarch64 targets
```

Comment:
Hmm, I feel like we should expand this. Having the first non-x86 platform reaching Tier 1 is a huge deal, and a ton of people are interested in support for the new Apple M1. Possible sketch of what content to put in:

* 64-bit ARM on Linux reached Tier 1 support
  * Every change done to the compiler now requires the full test suite to pass on the target.
  * First non-x86 target to be promoted to Tier 1!
  * Maybe: where the target could be used
* 64-bit ARM on Windows and Apple Silicon reached Tier 2 support!
  * Pre-built compilers are now guaranteed to be available through rustup
  * We're not running the compiler's test suite on those platforms, so there could be bugs in the compiler
  * Maybe: where the targets could be used

Reply:
You had mentioned really wanting these to be two sections, is there a reason why? they feel like the same thing to me, though I do think expanding on it would be good.

Reply:
Mostly because the guarantees provided by Tier 1 and Tier 2 are so different that merging them together might confuse people. Also I kinda want to give the first non-x86 target the spotlight on its own :D